requirements/edx/base.txt

```diff
@@ -18,21 +18,20 @@
 -e openedx/core/lib/xblock_builtin/xblock_discussion  # via -r requirements/edx/local.in
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@2d176468e33c0713c911b563f8f65f7cf232f5b6#egg=xblock-google-drive  # via -r requirements/edx/github.in
 -e common/lib/xmodule  # via -r requirements/edx/local.in
-amqp==1.4.9               # via kombu
+amqp==2.6.1               # via kombu
 analytics-python==1.2.9   # via -r requirements/edx/base.in
 aniso8601==8.0.0          # via edx-tincan-py35
-anyjson==0.3.3            # via kombu
 appdirs==1.4.4            # via fs
 attrs==20.2.0             # via -r requirements/edx/base.in, edx-ace
 babel==2.8.0              # via -r requirements/edx/base.in, enmerkar, enmerkar-underscore
 beautifulsoup4==4.9.1     # via pynliner
-billiard==3.3.0.23        # via celery
-bleach==3.2.0             # via -r requirements/edx/base.in, django-wiki, edx-enterprise, lti-consumer-xblock, ora2, xblock-poll
+billiard==3.6.3.0         # via celery
+bleach==3.1.5             # via -r requirements/edx/base.in, django-wiki, edx-enterprise, lti-consumer-xblock, ora2, xblock-poll
```

Comment:
Huh, it's not clear why this was downgraded.

Reply:
I will run make upgrade again and rebase it with master

Reply:
Updated